dist/tools/doccheck/check.sh

```diff
@@ -32,7 +32,7 @@ then
 fi
 
 exclude_filter() {
-    grep -v -e vendor -e examples -e tests
+    grep -e vendor -e examples -e tests -e "\<dist/tools\>"
```

Comment:
why remove `-v` ?

Reply:
Leftover from my experimentations with this script. I noticed it already when retesting, but forgot to put it back.

Reply:
Fixed and squashed